### DISCUSSION TITLE
[Feature Request] Code modification suggestions?

### DISCUSSION THREAD

**Discussion by zjplab** [community/community#20484] [Copilot Conversations]
Copilot is usually triggered only when you intend to write new lines of code or have written half of the line. Sometimes you copy and paste some code into the IDE and want  to modify later, which is a frequent use case. I think copilot can still be of value if it can auto-suggest some code modifications that can let you choose.  So could you add this feature as well. Thanks!

**Comment by Zoltus** [comment]
Not sure if i understood right, but you can see copilot suggested completions in the github copilot panel.
